articles/active-directory-b2c/partner-nok-nok.md

```diff
@@ -12,54 +12,56 @@ ms.date: 06/21/2024
 ms.author: gasinh
 ms.subservice: b2c
 
-# Customer intent: I'm a developer integrating Azure Active Directory B2C with a third-party authentication provider. I want to learn how to configure Nok Nok Passport as an identity provider (IdP) in Azure AD B2C. My goal is to enable passwordless FIDO authentication for my users.
+# Customer intent: I'm a developer integrating Azure Active Directory B2C with a third-party authentication provider. I want to learn how to configure Nok Nok S3 Authentication Suite as an identity provider (IdP) in Azure AD B2C. My goal is to implement FIDO Passkey authentication for my users.
 ---
-# Tutorial: Configure Nok Nok Passport with Azure Active Directory B2C for passwordless FIDO2 authentication
+# Tutorial: Configure Nok Nok S3 Authentication Suite with Azure Active Directory B2C for FIDO Passkey Authentication
 
-Learn to integrate the Nok Nok S3 Authentication Suite into your Azure Active Directory B2C (Azure AD B2C) tenant. The Nok Nok solutions enable FIDO certified multi-factor authentication such as FIDO UAF, FIDO U2F, WebAuthn, and FIDO2 for mobile and web applications. Nok Nok solutions improve security posture while balancing user the experience.
+Learn to integrate the Nok Nok S3 Authentication Suite into your Azure Active Directory B2C (Azure AD B2C) tenant. The Nok Nok solution enables FIDO-certified multifactor authentication such as FIDO UAF, FIDO U2F, WebAuthn, and FIDO2 for mobile and web applications. The Nok Nok solution strengthens your security while maintaining an optimal with user experience.
 
 Go to noknok.com to learn more: [Nok Nok Labs, Inc.](https://noknok.com/)
 
 ## Prerequisites
 
 To get started, you need:
 
-* An Azure subscription
-  * If you don't have one, get a [Azure free account](https://azure.microsoft.com/free/)
-* An Azure AD B2C tenant linked to the Azure subscription
-  * [Tutorial: Create an Azure Active Directory B2C tenant](tutorial-create-tenant.md) 
-* Go to [noknok.com](https://noknok.com/).
-  * On the top menu, select **Demo**.
+* An Azure subscription. If you don't have one, get an [Azure free account](https://azure.microsoft.com/free/)
+* An Azure AD B2C tenant linked to the Azure subscription [Tutorial: Create an Azure Active Directory B2C tenant](tutorial-create-tenant.md)
+* Nok Nok Cloud evaluation tenant to be used for registration and authentication

```

Comment:
Suggested revision:
* Use Nok Nok Cloud evaluation for registration and authentication.